reflex/config.py

```diff
@@ -219,6 +219,12 @@ class Config:
     # Number of gunicorn workers from user
     gunicorn_workers: Optional[int] = None
 
+    # Maximum expiration lock time for redis state manager
+    redis_state_manager_lock_expiration: int = constants.Expiration.LOCK
+
+    # Token expiration time for redis state manager
+    redis_state_manager_token_expiration: int = constants.Expiration.TOKEN
```

Comment:
```suggestion
    # Maximum expiration lock time for redis state manager
    redis_lock_expiration: int = constants.Expiration.LOCK

    # Token expiration time for redis state manager
    redis_token_expiration: int = constants.Expiration.TOKEN
```

lets use more concise names